internal/app/machined/pkg/controllers/block/internal/volumes/format.go

```diff
@@ -148,6 +158,14 @@ func GrowFilesystem(logger *zap.Logger, volumeContext ManagerContext) error {
 			return fmt.Errorf("error growing XFS: %w", err)
 		}
 
+		return nil
+	case block.FilesystemTypeEXT4:
+		logger.Info("growing ext4 filesystem", zap.String("device", volumeContext.Status.MountLocation))
+
+		if err := makefs.Ext4Resize(volumeContext.Status.MountLocation); err != nil {
```

Comment:
resize2fs works on the blockdevice, not the mount as like xfs

Reply:
yes, this is how it works here - on the blockdevice (it doesn't do any mounts)

Reply:
ahh okay, i assumed `volumeContext.Status.MountLocation` was the mounted location

Reply:
it's the mount source, e.g. for encrypted devices it's different from "Location" which is the encrypted blockdevice